### PR TITLE
Menu hotfix

### PR DESCRIPTION
**Pull request by ValentinaPieri**

### CODE REVIEW

**Review by benedettapacilli — APPROVED** [review]

**Review by criicchio — APPROVED** [review]

**Review by expl0r3rgu1 — CHANGES_REQUESTED** [review]
Waiting for the new demo testing fixes

**Review by criicchio — APPROVED** [review]

File: src/main/menu/MenuPanel.java [hunk 1]
```diff
@@ -184,12 +180,12 @@ private void addButton(String name, int gridx, int gridy, int ipadx, int ipady,
 	 */
 	private void addImage(String name, int gridx, int gridy, int ipadx, int ipady, int top, int left, int bottom,
 			int right) {
-		this.add(this.getImageResource(name),
+		this.add(CommonMethods.getJLabelImage(name, 20),
 				new GBCSimplified(gridx, gridy, CommonMethods.getPixelsFromPercentageWidth(ipadx),
 						CommonMethods.getPixelsFromPercentageHeight(ipady),
-						new Insets(CommonMethods.getPixelsFromPercentageHeight(top),
+						new Insets(CommonMethods.getPixelsFromPercentageWidth(top),
```
Comment (expl0r3rgu1) [inline src/main/menu/MenuPanel.java:186]: Replace with: 
```java
CommonMethods.getPixelsFromPercentageHeight(top)
```

File: src/main/menu/MenuPanel.java [hunk 2]
```diff
@@ -184,12 +180,12 @@ private void addButton(String name, int gridx, int gridy, int ipadx, int ipady,
...
 			int right) {
-		this.add(this.getImageResource(name),
+		this.add(CommonMethods.getJLabelImage(name, 20),
 				new GBCSimplified(gridx, gridy, CommonMethods.getPixelsFromPercentageWidth(ipadx),
 						CommonMethods.getPixelsFromPercentageHeight(ipady),
-						new Insets(CommonMethods.getPixelsFromPercentageHeight(top),
+						new Insets(CommonMethods.getPixelsFromPercentageWidth(top),
 								CommonMethods.getPixelsFromPercentageWidth(left),
-								CommonMethods.getPixelsFromPercentageHeight(bottom),
+								CommonMethods.getPixelsFromPercentageWidth(bottom),
```
Comment (expl0r3rgu1) [inline src/main/menu/MenuPanel.java:188]: Replace with:
```java
CommonMethods.getPixelsFromPercentageHeight(top)
```

**Review by expl0r3rgu1 — CHANGES_REQUESTED** [review]
Row 158 of SelectionPanel class Use height and width properly

**Review by expl0r3rgu1 — APPROVED** [review]

**Review by benedettapacilli — APPROVED** [review]

**Pull request merged**